EasyPost.Tests/EasyPostTest.cs

```diff
@@ -1,62 +1,21 @@
-using System.Dynamic;
-using System.Threading.Tasks;
-using EasyPost.Utilities;
-using Microsoft.VisualStudio.TestTools.UnitTesting;
-using RestSharp;
+
+using Xunit;
 
 namespace EasyPost.Tests
 {
-    [TestClass]
     public class EasyPostTest
     {
         private const string FakeApikey = "fake_api_key";
-        private const string FakeApiUrl = "https://fake.api.com";
-        private const string HttpBinUrl = "https://httpbin.org/get";
 
-        private static ClientConfiguration GetBasicClientConfiguration()
-        {
-            return new ClientConfiguration(FakeApikey);
-        }
-
-        [TestMethod]
-        public void TestApiKeyConstructor()
-        {
-            ClientConfiguration config = GetBasicClientConfiguration();
-
-            Assert.AreEqual(FakeApikey, config.ApiKey);
-            Assert.AreEqual("https://api.easypost.com/v2", config.ApiBase);
-        }
-
-        [TestMethod]
-        public void TestApiKeyPlusBaseUrlConstructor()
-        {
-            ClientConfiguration config = new ClientConfiguration(FakeApikey, FakeApiUrl);
-
-            Assert.AreEqual(FakeApikey, config.ApiKey);
-            Assert.AreEqual(FakeApiUrl, config.ApiBase);
-        }
-
-        [TestMethod]
+        [Fact]
         public void TestTimeout()
         {
-            Client client = new Client(GetBasicClientConfiguration());
+            Client client = new Client(FakeApikey);
             client.ConnectTimeoutMilliseconds = 5000;
             client.RequestTimeoutMilliseconds = 5000;
 
-            Assert.AreEqual(5000, client.ConnectTimeoutMilliseconds);
-            Assert.AreEqual(5000, client.RequestTimeoutMilliseconds);
-        }
-
-        [TestMethod]
-        public async Task TestClientManagerGetCurrent()
```

Comment:
Question/Issue: Why were all of these tests removed? We should continue allowing users to override the base URL. This is important for various reasons - our internal teams override the base URL during E2E tests to hit their Homeowner instances when doing internal tests, and some users may way to hit a mock server for their own testing. If we have removed the ability for users to override this value, we need to adjust the client object to continue accepting this parameter.

Reply:
I was unaware that was a requirement of our library. I will introduce that capability.

Reply:
Thanks! I believe most of our libs allow for this and thought that's what this test was testing.

Can we please add back in the tests that check if we can override the base URL?

Reply:
Resolved.